docs/topics/jvm/jvm-get-started.md

```diff
@@ -32,26 +32,27 @@ To get started, first download and install the latest version of [IntelliJ IDEA]
 7. Enable the **Add sample code** option to create a file with a sample `"Hello World!"` application.
 8. Click **Create**.
 
-## Create an application
-
-1. If you chose the Gradle build system, you have `build.gradle(.kts)`, a build script file, in your project. It includes
-   the `kotlin("jvm")` plugin and dependencies required for your console application. Ensure that you use the latest
-   version of the plugin:
+  > If you chose the Gradle build system, you have in your project a build script file: `build.gradle(.kts)`. It includes 
+  > the `kotlin("jvm")` plugin and dependencies required for your console application. Make sure that you use the latest 
+  > version of the plugin:
+  > 
+  > ```kotlin
+  > plugins {
+  >     kotlin("jvm") version "%kotlinVersion%"
+  >     application
+  > }
+  > ```
+  >{type="note"}
 
-   ```kotlin
-   plugins {
-       kotlin("jvm") version "%kotlinVersion%"
-       application
-   }
-   ```
+## Create an application
 
-2. Open the `main.kt` file in `src/main/kotlin`.  
-   The `src` directory contains Kotlin source files and resources. The `main.kt` file contains sample code that will print 
+1. Open the `Main.kt` file in `src/main/kotlin`.  
```

Comment:
The wizard creates the file `Main.kt` with a capital letter. Please revert these changes

Reply:
And I corrected it to a capital letter. Then what's the problem?

Reply:
Oh, I'm sorry, I thought it's vice versa. It's a common error in our documentation